article/app/liveblog/LiveBlogCurrentPage.scala

```diff
@@ -1,10 +1,71 @@
 package liveblog
 
-import model.liveblog.BodyBlock
+import model.liveblog.{Blocks, BodyBlock}
+
+sealed trait BlockRange { def query: Option[Seq[String]] }
+case object Canonical extends BlockRange {
+  val firstPage = "body:latest:30"
+  val oldestPage = "body:oldest:1"
```

Comment:
just need to get any block id in the last page so we can work out the link for the navigation bar when you click "oldest"